pkg/karmadactl/apply.go

```diff
@@ -86,6 +95,7 @@ func NewCmdApply(karmadaConfig KarmadaConfig, parentCommand string) *cobra.Comma
 	o.KubectlApplyFlags.AddFlags(cmd)
 	cmd.Flags().StringVarP(&o.Namespace, "namespace", "n", o.Namespace, "If present, the namespace scope for this CLI request")
 	cmd.Flags().BoolVarP(&o.AllClusters, "all-clusters", "", o.AllClusters, "If present, propagates a group of resources to all member clusters.")
+	cmd.Flags().StringSliceVarP(&o.Clusters, "cluster", "C", o.Clusters, "If present, propagates a group of resources to the specified cluster.")
```

Comment:
```suggestion
	cmd.Flags().StringSliceVarP(&o.Clusters, "clusters", "C", o.Clusters, "If present, propagates a group of resources to the specified clusters.")
```

Reply:
@lonelyCZ 

I prefer to `--cluster` rather than `--clusters`. because `--clusters` make users confused. for example,

```sh
karmadactl apply -f manifests.yaml --clusters member1 --clusters member
```

what do you think ?

Reply:
FYI: https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/create/create_clusterrole.go#L96

For the parameters of the array type, the flag is generally in the singular form, not the plural form.

Reply:
> ```
> karmadactl apply -f manifests.yaml --clusters member1 --clusters member
> ```

Generally, we do not to use `--clusters` twice.

> FYI: https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/create/create_clusterrole.go#L96
>
> For the parameters of the array type, the flag is generally in the singular form, not the plural form.

Nice finding, I agree with you.

suggest for the example

```
%[1]s apply -f manifest.yaml --cluster member1,member2
```